mobile_base_sdk/mobile_base_sdk.py

```diff
@@ -162,12 +155,15 @@ def goto(
         The tolerance represents the margin along x, y and theta for which we consider
         that the mobile base has arrived its goal.
         """
+        if self.is_off():
```

Comment:
The formatting difference makes it hard to read but I guess this is the only thing to review here